### PR TITLE
Added summary of what can be done with the ARM API.

### PR DESCRIPTION
**Pull request by hannanelson**

### CODE REVIEW

**Review by eketabch — APPROVED** [review]

File: modules/ROOT/pages/runtime-manager-api.adoc [hunk 1]
```diff
@@ -1,15 +1,52 @@
-= About the Runtime Manager REST API
+= Runtime Manager API
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords: cloudhub, cloudhub api, manage, api, rest
 
-The Runtime Manager REST API enables you to programmatically access much of the functionality provided by Runtime Manager. To deploy Mule applications using the Runtime Manager REST API, see https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/arm-rest-services[REST APIs].
+The Runtime Manager REST API enables you to programmatically access these functions of Runtime Manager:
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-manager-api.adoc:7]: Are you sure you want to remove "REST" from the topic title? Or is there no other type of RM API?

File: modules/ROOT/pages/runtime-manager-api.adoc [hunk 2]
```diff
@@ -1,15 +1,52 @@
...
+** Add or remove a server from a cluster.
+** Modify clustering information for a server.
+** Delete a cluster.
+
+* Manage app deployments on your servers:
++
+** Deploy, redeploy, and undeploy apps.
+** Get application deployment and status information.
+** Manage flows, schedulers, and alerts in a deployed app.
+
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-manager-api.adoc:42]: Such nice, clear, simple, parallel list items! Bless you.

File: modules/ROOT/pages/runtime-manager-api.adoc [hunk 3]
```diff
@@ -1,15 +1,52 @@
...
+For an interactive reference of all the operations that are supported through the API, see https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/arm-rest-services[Runtime Manager REST API].
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-manager-api.adoc]: Oh...is this why the title above does not include REST?

File: modules/ROOT/pages/runtime-manager-api.adoc [hunk 4]
```diff
@@ -1,15 +1,52 @@
...
+
+The Runtime Manager REST API only manages on-prem applications deployed to your servers. To manage apps deployed to CloudHub using the API, see https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/cloudhub-api/[CloudHub API].
```
Comment (kbostrom) [inline modules/ROOT/pages/runtime-manager-api.adoc]:
```suggestion
The Runtime Manager REST API manages only on-prem applications deployed to your servers. To manage apps deployed to CloudHub using the API, see https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/cloudhub-api/[CloudHub API].
```

**Review by kbostrom — APPROVED** [review]
Done

File: modules/ROOT/pages/runtime-manager-api.adoc (hunk 2 above)
Comment (hannanelson) [reply modules/ROOT/pages/runtime-manager-api.adoc:42]: :)

File: modules/ROOT/pages/runtime-manager-api.adoc (hunk 1 above)
Comment (hannanelson) [reply modules/ROOT/pages/runtime-manager-api.adoc:7]: Added back - it wasn't in the TOC, but it is on Exchange.

**Pull request merged**